modules/renter/hostdb/hostdb.go

```diff
@@ -58,5 +118,7 @@ func New() *HostDB {
 	}
 	go hdb.threadedScan()
 
-	return hdb
+	cs.ConsensusSetSubscribe(hdb)
```

Comment:
clarification: the renter doesn't subscribe. It only needs `cs` to call `cs.Height`.